docs/_layouts/specifications.html

```diff
@@ -27,7 +27,8 @@ <h1 class="pr-32 pb-16 pt-32">{{ page.title }}</h1>
                     </div>
                     <div class="table-of-contents w-full md:w-1/3 md:sticky md:top-32 pb-12 md:pl-12">
                       {% include versions-dropdown.html %}
-                        <div class="rounded-lg p-4 border border-gray-400">
+                      {% include status.html %}
+                        <div class="rounded-lg p-4 border border-gray-400 mt-4">
```

Comment:
No need for the `mt-4`. Can revert to the original version.

```suggestion
                        <div class="rounded-lg p-4 border border-gray-400">
```